Code/Tools/AssetProcessor/native/resourcecompiler/RCQueueSortModel.cpp

```diff
@@ -197,21 +197,11 @@ namespace AssetProcessor
         {
             return priorityLeft > priorityRight;
         }
-
-        // Optionally stabilize queue order on the source name.
-        // This is used in automated tests, to allow tests to have a stable
-        // order that jobs with otherwise equal priority run, so tests process
-        // assets in the same order each time they are run.
-        if (m_sortQueueOnDBSourceName)
-        {
-            return leftJob->GetJobEntry().m_databaseSourceName < rightJob->GetJobEntry().m_databaseSourceName;
-        }
         
         // if we get all the way down here it means we're dealing with two assets which are not
         // in any compile groups, not a priority platform, not a priority type, priority platform, etc.
         // we can arrange these any way we want, but must pick at least a stable order.
-
-        return leftJob->GetJobEntry().m_jobRunKey < rightJob->GetJobEntry().m_jobRunKey;
+        return leftJob->GetJobEntry().m_databaseSourceName < rightJob->GetJobEntry().m_databaseSourceName;
     }
```

Comment:
I actually recommend keeping the run key sort there, in case source name is the same.  Which can happen for 2 jobs same source, right? You can compare the source name. first, then if that fails, fallback to prior behavior

Reply:
gotcha, I've updated it to that